Makefile

```diff
@@ -4,8 +4,8 @@
 #
 
 #Magic hackery to allow shell envs to be set in Makefile
-IGNORE := $(shell bash -c "source set_env.sh; env | sed 's/=/:=/' | sed 's/^/export /' > makeenv")
-include makeenv
+#IGNORE := $(shell bash -c "source set_env.sh; env | sed 's/=/:=/' | sed 's/^/export /' > makeenv")
+#include makeenv
```

Comment:
Was indeed planning that, only reason I commented out was to have more commits (more chance of hitting flakeyness) . .